### PR TITLE
STAC-9832: Add traces perspective documentation

### PR DESCRIPTION
**Pull request by lmreis89**

### CODE REVIEW

**Comment by lmreis89** [comment]
Still needs documentation on configuration of traces.

**Comment by gggina** [comment]
Added traces configuration page

File: use/perspectives/trace-perspective.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+When you are in a view, you fetch traces for the components in that view. These same topology filters can be used in the trace perspective to reduce or increase the number of components for which you want to fetch traces.
+
+* Learn more about how to use topology filters in the [topology perspective page](topology-perspective.md).
+* Find out [how traces and topology](trace-perspective.md#traces-and-topology) are related.
+
+### Trace Filters
+
+A trace can be filtered based on two properties of its spans: span types and span tags. The image below shows the filters menu where you see filters for topology, events and traces:
+
+![The traces perspective and its filters](../../.gitbook/assets/trace-filters.png)
```
Comment (mpvvliet) [inline use/perspectives/trace-perspective.md:44]: I think we are calling it "trace perspective"

File: use/perspectives/trace-perspective.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+For example, if you filter the trace list for all spans of type **database**, this will return all traces that have at least one span whose type is **database**.
+
+## Traces and Topology
+
+In StackState, a [view](../views.md) shows you a sub-selection of your IT infrastructure in terms of components and relations. A number of our supported integrations send traces to StackState via [our agent](../../configure/how_to_setup_traces.md). These traces are used in the traces perspective and also in the [topology perspective](topology-perspective.md) to create the topology of your view.
```
Comment (mpvvliet) [inline use/perspectives/trace-perspective.md:50]: Same here, is it "traces perspective" or "trace perspective"?

File: use/perspectives/trace-perspective.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+When ingesting traces, StackState attaches service identifiers to the components that are created. These identifiers are also included as part of the **service** property of the spans in a trace. All topology created out of a trace will have a tag **has_traces**, this allows you to easily identify components for which you have traces.
+
+## Sorting and Limiting
+
+As of version 4.1, all traces are sorted by latency (descending). This is the only sorting criteria available in this version. The trace list is not limited by size, you can scroll infinitely to see all traces that are available for your component, filter and time selections. Finally, on top of the list of traces, StackState displays an approximation of the total amount of traces that are returned from the filters you have selected.
+
+## Time Traveling Considerations
+
+When using the *trace perspective*, just like in other perspectives, you can either be in live mode or in the past. In live mode, StackState is constantly polling for new traces. When time traveling to the past, all views in all perspectives come with a timeline for which you can make two selections:
```
Comment (mpvvliet) [inline use/perspectives/trace-perspective.md:77]: Are we always using italics to refer to the trace perspective? I've seen regular (non-italic) text as well.

Comment (gggina) [reply use/perspectives/trace-perspective.md:77]: I checked the StS ui, there we capitalize the perspective names (e.g. Topology Perspective) and we also use 'Traces Perspective' in place of 'trace perspective'. I will update to match this and remove italics.

**Review by gggina — APPROVED** [review]
addressed comments from Martin

**Pull request merged**